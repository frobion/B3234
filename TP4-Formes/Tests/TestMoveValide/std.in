R Rectangle1 1 3 6 0
MOVE Rectangle1 5 6
LIST
EXIT

