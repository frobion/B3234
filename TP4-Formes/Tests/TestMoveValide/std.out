OK
OK
R Rectangle1 6 9 11 6


